W-14493510-rpamanagerbotupdate-se (#319)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

* If you want to assign process applications to a bot, you need to create them in the *Process Applications* view of the *Process Automation* module first.
* Ask an organization administrator in Access Management to assign you the required permissions:
+
[%header, cols="1,1,1"]
|===
|Action |RPA Permission |Deprecated RPA Permission

> |Open the *Bot Management* module and view the bots.

Reviewer: (Globally) According to our style guide, if one sentence has a full stop, all the sentences in the table must take full stop. Keep consistency in the doc.

Author: I did that for all tables with commit [8fb8a2a](https://github.com/mulesoft/docs-rpa/pull/319/commits/8fb8a2a213f523731774b13f6921433ad946ebee).

|RPA Bots Manager, RPA Operations Viewer, RPA Operations Manager, RPA Administrator
|Bot Management Open

> |License, edit, update, and remove bots

Reviewer 2: ```suggestion
|License, edit, update, and remove bots.
```

Reviewer 2: (terminating punctuation for both action rows, since they're complete sentences.)

Reviewer 2: For the heading, instead of adding bold to the heading columns, use the header option to make first row a heading.

(Items in bold are not translated.)

Author: Done for all tables in RPA Manager docs

. In the form *<bot name>* click on all service times and applications you want to assign.
. Click *Save*.

== Update a Bot

> Update your bots to the latest version to get the newest features, bugfixes, and security enhancements. The bot finishes all processes before updating, and resumes them afterwards to avoid data loss.

Reviewer: ```suggestion
Update your bots to the latest version to get the newest features, bug fixes, and security enhancements. The bot finishes all processes before updating and resumes them afterwards to avoid data loss.
```

> Update your bots to the latest version to get the newest features, bugfixes, and security enhancements. The bot completes all processes before updating, and resumes them afterwards to avoid data loss.

Reviewer 2: Make sure this terminology (completes processes) is the same in the UI text.

Author: replaced with finishes to match UI text

Alternatively, you can update single bots manually by downloading and installing the newer version from the *Download Applications* pane in the *Home* module of RPA Manager.

> You can automatically update bots that fullfill the following requirements:

Reviewer 2: ```suggestion
You can automatically update bots that fulfill the following requirements:
```

* The bot is in OK status.

> * The bot is version 1.5 or later.

Reviewer: ```suggestion
* The bot version is 1.5 or later.
```

> * The bot is of version 1.5 or later.

Reviewer 2: ```suggestion
* The bot version is 1.5 or later.
```
